packages/tldraw/src/lib/ui/components/primitives/TldrawUiButtonPicker.tsx

```diff
@@ -23,6 +22,7 @@ export interface TLUiButtonPickerProps<T extends string> {
 	items: StyleValuesForUi<T>
 	theme: TLDefaultColorTheme
 	onValueChange(style: StyleProp<T>, value: T): void
+	onHistoryMark?(id: string): void
```

Comment:
These components no longer depend on the editor, so I've extracted the history-effecting things and made them optional

Reply:
They feel quite specific to tldraw so it feels like we might keep the editor. Or do we have some other uses where editor is not available?